index.html

```diff
@@ -2,31 +2,44 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="canonical" href="http://httpsters.me">
     <title>We are httpsters</title>
-    <style>
+    <!-- dude, not all of us are httpsters, some of us are still httpters -->
+    <!-- (it's called let's encrypt and it's free!!) -->
+    <!-- [techincally we're not https so...] -->
+    
+    <style type="text/css">
     body {
         background:lightgrey;
         font-size:18px;
         padding:20px;
     }
 
+    a img {
+        border: none;
+    }
+
     </style>
+
 </head>
-<body>
+
+<body class="hello-world">
     <h1>Welcome to httpsters.me</h1>
+   
     A collaboration made possible by:
     <ul>
-        <li><a href="http://github.com/davidnuon">David Nuon</a></li>
-        <li><a href="https://github.com/ryan-b-w">Ryan Wilcox</a></li>
-        <li><a href="https://github.com/masterkale">Matthew Miller</a></li>
-        <li><a href="https://github.com/postphotos">Leo Postovoit</a></li>
+        <li><a href="http://github.com/davidnuon" title="@davidnuon">David Nuon</a></li>
+        <li><a href="https://github.com/ryan-b-w" title="@ryan-b-w">Ryan Wilcox</a></li>
+        <li><a href="https://github.com/masterkale" title="@masterkale">Matthew Miller</a></li>
+        <li><a href="https://github.com/postphotos" title="@postphotos">Leo Postovoit</a></li>
     </ul>
-    <hr>
+    <hr id="horizontal-rule">
 
-    <img src="img/arfie.gif" alt="Arfie" title='Arfie says: "Comments and Suggestions Are Welcome"' >
-    <img src="img/netscape.gif" alt="Best viewed with Netscape Navigator" >
-    <img src="img/vim.gif" alt="Created with vim with &heart;" >
-    <img src="img/made-with-emacs.png" alt="Made with Emacs" >
+    <a href="http://www.vintagekennelclub.com/"><img src="img/arfie.gif" alt="Arfie" title='Arfie says: "Comments and Suggestions Are Welcome"' ></a>
```

Comment:
This needs to link to dogpile.com, the search engine to which Arfie is the official mascot 🐶 

https://web.archive.org/web/19970710110326im_/http://www.dogpile.com:80/

Reply:
I think that is fantastic 💟 🐶

Reply:
Let's be real though - vintagekennelclub.com is quite funny too